microsoft-edge/webview2/concepts/print.md

```diff
@@ -0,0 +1,736 @@
+    DisplayPdfDataInPrintDialog(stream);
+
+    MessageBox.Show(this, "Printing " + title + " document to PDF Stream " + ((stream != null) ? "succedded" : "failed"), "Print To PDF Stream");
+  }
+  catch(Exception exception)
+  {
+    MessageBox.Show(this, "Printing to PDF Stream failed: " + exception.Message, "Print to PDF Stream");
+  }
+}
+
+// Function to display current web page pdf data in a custom print preview dialog.
+void DisplayPdfDataInPrintDialog(Stream pdfData)
+{
+  // You can display the printable pdf data in a custom print preview dialog to the end user.
+}
+```
+
+**APIs:**
+
+* [CoreWebView2.PrintToPdfStreamAsync(printSettings) Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.printtopdfstreamasync#microsoft-web-webview2-core-corewebview2-printtopdfstreamasync)
+
+
+<!-- ------------------------------ -->
+
+# [C++](#tab/cpp)
+
+```cpp
+static void DisplayPdfDataInPrintDialog(IStream* pdfData)
+{
+  // You can display the printable pdf data in a custom print preview dialog to the end user.
+}
+
+bool AppWindow::PrintToPdfStream()
+{
+  wil::com_ptr<ICoreWebView2_16> webView2_16;
+  CHECK_FAILURE(m_webView->QueryInterface(IID_PPV_ARGS(&webView2_16)));
+
+  wil::unique_cotaskmem_string title;
+  CHECK_FAILURE(m_webView->get_DocumentTitle(&title));
+
+  // Passing nullptr for `ICoreWebView2PrintSettings` results in default print settings used.
+  CHECK_FAILURE(
+      webView2_16->PrintToPdfStream(
+          nullptr,
+          Callback<ICoreWebView2PrintToPdfStreamCompletedHandler>(
+              [title = std::move(title), this](HRESULT errorCode, IStream* pdfData) -> HRESULT
+              {
+                DisplayPdfDataInPrintDialog(pdfData);
+                std::wstring message = L"Printing " + std::wstring(title.get()) +
+                                       L" document to PDF Stream " +
+                                       ((errorCode == S_OK && pdfData != nullptr) ? L"succedded" : L"failed");
+
+                AsyncMessageBox(message, L"Print to PDF Stream");
+
+                return S_OK;
+              })
+              .Get()));
+  return true;
+}
+```
+
+**APIs:**
+
+* [ICoreWebView2_16::PrintToPdfStream(printSettings, handler) method](/microsoft-edge/webview2/reference/win32/icorewebview2_16#printtopdfstream)
+
+---
+
+<!-- end of tab-set -->
+
+
+<!-- ====================================================================== -->
+## API Reference overview
+
+
+<!-- ------------------------------ -->
+
+# [C#](#tab/csharp)
+
+* [CoreWebView2.ShowPrintUI() Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.showprintui#microsoft-web-webview2-core-corewebview2-showprintui)
+* [CoreWebView2.ShowPrintUI(CoreWebView2PrintDialogKind) Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.showprintui#microsoft-web-webview2-core-corewebview2-showprintui(microsoft-web-webview2-core-corewebview2printdialogkind))
```

Comment:
```suggestion
* [CoreWebView2.ShowPrintUI(printDialogKind) Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.showprintui#microsoft-web-webview2-core-corewebview2-showprintui(microsoft-web-webview2-core-corewebview2printdialogkind))
```
https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2.showprintui?view=webview2-dotnet-1.0.1549-prerelease#microsoft-web-webview2-core-corewebview2-showprintui(microsoft-web-webview2-core-corewebview2printdialogkind)

Reply:
Added to show overloaded method in dotnet.